pkgs/development/python-modules/pytest-flask/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest, flask, setuptools_scm }:
+
+buildPythonPackage rec {
+  pname = "pytest-flask";
+  version = "0.15.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0jdzrib94vwfpl8524h34aqzqndh3h4xn706v32xh412c8dphx6q";
+  };
+
+  doCheck = true;
+
+  propagatedBuildInputs = [
+    pytest
```

Comment:
Please make this a `buildInput` so we can allow packages to depend on a different version of pytest.